src/persist/zcl_abapgit_persistence_repo.clas.abap

```diff
@@ -83,6 +83,10 @@ CLASS ZCL_ABAPGIT_PERSISTENCE_REPO IMPLEMENTATION.
     IF sy-subrc = 0.
       rs_repo-local_settings-ignore_subpackages = abap_true.
     ENDIF.
+    FIND FIRST OCCURRENCE OF '<SERIALIZE_MASTER_LANG_ONLY>X</SERIALIZE_MASTER_LANG_ONLY>' IN lv_xml.
+    IF sy-subrc = 0.
+      rs_repo-local_settings-main_language_only = abap_true.
+    ENDIF.
```

Comment:
add a comment here? that its for automatic conversion, and can be removed on, guess 2023-01-01?